diff --git a/dist/chunk-XNEM3CPC.js b/dist/chunk-XNEM3CPC.js
index d6becdbad7770db231bf3f7cc314697ea7803494..4b011496cb88556ff28132427aafcfb4598aa660 100644
--- a/dist/chunk-XNEM3CPC.js
+++ b/dist/chunk-XNEM3CPC.js
@@ -1,4 +1,6 @@
-import { nodes, Tag } from '@markdoc/markdoc';
+import Markdoc from '@markdoc/markdoc';
+const { nodes: defaultNodes, Tag } = Markdoc;
+const nodes = defaultNodes;
 
 // src/childText.ts
 function childText(node) {
